src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,6 +54,14 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
+        try {
+            if (dateParams.length == 0) {
```

Comment:
I get your idea here, but let's remove this checking and add ArrayIndexOutOfBoundsException to catch block